oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -76,7 +79,6 @@
      * @since 7.0.0 dayStep represents how many days a single one index represents. Default is 1, meaning no difference
      * with previous versions. But if there isn't much traffic for single one day, user could set the step larger to
      * reduce the number of indexes, and keep the TTL longer.
-     *
```

Comment:
These lines should not be re-formated.

Reply:
When I add this line, it will still be cleared after formatting with skywalking code style